docs/source/resources/tutorial-template.rst

```diff
@@ -0,0 +1,81 @@
+=================
+Tutorial template
+=================
+
+.. contents::
+   :local:
+   :depth: 1
+   
+.. _documentation@anaconda.com: documentation@anaconda.com
+
+*This document describes the steps for creating*
+*a tutorial for conda build*.
+
+* *Follow the* :doc:`writing style guide <style-guide>`.
+* *Copy the template*: https://github.com/conda/conda-build/tree/master/docs/source/resources/tutorial-template.rst.
```

Comment:
Consider putting the template copy step first, and combining the 1st & 3rd into something like "Replace the italicized text with your tutorial content, following the writing style guide.